xmovie/presets.py

```diff
@@ -189,6 +194,9 @@ def _set_style(fig, ax, pp, style):
 
 
 def _add_land(ax, style):
```

Comment:
See my comment above. It would be really neat, if we could just write:

```python
@needs_cartopy
def func(...):
````
here and in all the other components that need it.